app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -86,6 +87,16 @@ class BrowserFragment : Fragment() {
             engineView
         )
 
+        val findInPageIntegration = FindInPageIntegration(requireComponents.core.sessionManager, findInPageView)
+        val toolbarIntegration = ToolbarIntegration(
+                requireContext(),
+                toolbar,
+                requireComponents.toolbar.shippedDomainsProvider,
+                requireComponents.core.historyStorage
+            )
+
+        lifecycle.addObservers(findInPageIntegration, toolbarIntegration, sessionFeature)
```

Comment:
We have a `lifecycle.addObservers` for for these classes below. the `findInPageIntegration` should be added to the one below and this deleted.